.github/workflows/build.yml

```diff
@@ -0,0 +1,57 @@
+name: build
+
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+  workflow_dispatch:
+  
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      # required by aws-actions/configure-aws-credentials
+      id-token: write
+      contents: read
+    steps:
+    - name: run number with offset
+      env:
+        NUM: ${{ github.run_number }}
+      run: echo "GITHUB_RUN_NUMBER_WITH_OFFSET=$((10000+$NUM))" >> $GITHUB_ENV
```

Comment:
Is this needed? I'd have thought setting `buildNumberOffset` in `guardian/actions-riff-raff@v2` is sufficient here?